### PR TITLE
update /core to vbt

### LINKED ISSUE #1598: /core/ to VBT

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

* update /core to vbt
* created a pattern for logo clouds

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [page](http://0.0.0.0:8001/core)
- Compare to the [design](https://www.dropbox.com/work/00_web_team/_ubuntu.com/new%20projects/vanilla%20update/Patterns%20inventory/6.%20Core?preview=core-overview.png) and the [demo](http://www.ubuntu.com-1598-core-overview.demo.haus/core)

## Issue / Card

Fixes #1598

### CODE REVIEW

**Comment by pmahnke** [comment]
@barrymcgee because I thought I tried it and it was just a straight line of images... very small, but I think I might have tried p-inline-list-icons by mistake...

**Comment by pmahnke** [comment]
@barrymcgee I did the following:

* used p-inline-images
*added the missing __item css for now
* created a __images to constrain the image size for asset server logos

**Comment by pmahnke** [comment]
@barrymcgee updated with @deadlight 's `.p-inline-image update`

**Comment by barrymcgee** [comment]
On S screens, the image is nudging the preceding strip border
screen shot 2017-05-24 at 16 54 52

**Comment by pmahnke** [comment]
@barrymcgee updated the feature images.. thanks for the other fixes

**Comment by yaili** [comment]
Design review:
- [ ] Can the hero text block be 1col narrower so there's more space with the image?
- [ ] Let's use h4 styling in the 4 sections of "Why use Ubuntu Core?" 
- [ ] Can the Core logo vertically centre with the text next to it?
- [ ] Why is there inline style for the diagram "padding-top: 2em;" can we remove it?
- [ ] Should the "The smallest Ubuntu" be an h2?
- [ ] "Let’s work together" should be white and is missing top border
- [ ] I'm assuming all the non equal height boxes I see in demos have been fixed somewhere else, right?
- [ ] S screen: hero image is touching top of text block when stacked - needs space
- [ ] What can we do about the layout of the hero at exactly 768px - we seem to forget about this threshold in our media queries
- [ ] Can the Artik logo be bigger, does it have margin or something?

**Comment by pmahnke** [comment]
@yaili I fixed everything with two notes:

1. .p-divider equal height was forced with a u-equal-height class, which we will have to amend sometime
2. the 768px is likely a Vanilla grid / visibility bug? @barrymcgee ?

**Comment by yaili** [comment]
- [ ] There used to be a bigger gap between the intro paragraph and the 4 columns in "Why use Ubuntu Core?" now it's too close - do we have a way to separate an element from another easily? This happens a lot where things are too close to each other
- [ ] Artik logo still small, it has extra padding, do you want me to do a new one, or is it coming from partners?

**Comment by pmahnke** [comment]
@yaili I put the 4 in a new strip.. but inline removed the bottom padding on the top one... which might be solved another way more formally? and cropped the artik image

File: static/sass/_v1_pattern_link.scss [hunk 1]
```diff
@@ -12,9 +12,18 @@
     }
   }
 
-  .p-button--neutral {
-    span.p-link--external {
-      background-image: url('#{$assets-path}e1bba201-external-link-cool-grey.svg');
+  .p-button {
+    &--neutral {
+      span.p-link--external {
```
Comment (barrymcgee) [inline static/sass/_v1_pattern_link.scss]: This selector is over-qualified, please remove `span`

File: static/sass/_v1_pattern_link.scss [hunk 2]
```diff
@@ -12,9 +12,18 @@
...
+        background-image: url('#{$assets-path}e1bba201-external-link-cool-grey.svg');
+      }
+    }
+
+    &--brand {
+      span.p-link--external {
```
Comment (barrymcgee) [inline static/sass/_v1_pattern_link.scss]: This selector is over-qualified, please remove `span`

**Comment by pmahnke** [comment]
@barrymcgee fixed the span. s

**Pull request merged**